release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -2050,6 +2063,8 @@ You don't have any projects. You can try to create a new project, by running
 +
 This warning message is a known issue but does not indicate any functionality issues with {product-title}. You can safely ignore the warning message and continue to use {product-title} as intended. (link:https://issues.redhat.com/browse/OCPBUGS-44833[*OCPBUGS-44833*])
 
+* For dual-stack networking clusters that run on {rh-openstack}, when a Virtual IP (VIP) that is attached to a Floating IP (FIP) moves between master nodes, the association between VIP and FIP may stop working if the new master is on a different compute node. This issue occurs because OVN assumes that both IPv4 and IPv6 addresses on a shared Neutron port belong to the same node. (link:https://issues.redhat.com//browse/OCPBUGS-50599[*OCPBUGS-50599*])
```

Comment:
Just a heads-up that the bug is ON_QA and once it's verified and is part of the release, I guess in the first z-stream (planned for March 11), the known issue will need to be removed.